lang/en/tool_mergeusers.php

```diff
@@ -186,8 +186,15 @@
     data from new user (by default). This also means that data from old user is 
     deleted to keep the consistence. Otherwise, if you uncheck this option, 
     data from old user will be kept.';
-
 $string['starttime'] = 'Started merging at {$a}';
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['queue_to_process_pending_merge_requests'] = 'Get queue of the pending merge requests and launch adhoc task for each single request';
+$string['cannotfinduser'] = 'Cannot find the user. Change the input parameters.';
+$string['toomanyusers'] = 'Too many users found. Change the input parameters.';
+$string['maxattempts'] = 'Max number of attempts to merge users';
+$string['maxattempts_desc'] = 'Max number of attempts to merge users accounts. If the adhoc task reaches this number of attempts, 
```

Comment:
strings should be placed all in a single line (see core). It is a rule for the tool of managing translations in Moodle.

I would simplify the description, something like: `'This is the maximum number of attempts that a merge request will be tried to merge. After that number of attempts without success, the request will not be processed anymore and it will be updated as ended with errors.'`